### PR TITLE
chore: add readme for cutting release

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: tools/cut_a_release.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Cutting a Deno release
+
+**During this process `main` branch should be frozen and no commits should land
+until the release is cut.**
+
+1. Create a PR that bumps versions of all crates in `op_crates` and `runtime`
+   directories.
+
+If this is a Deno patch release then the crates should have patch version bumped
+as well; analogously for a minor Deno release the crates should have minor
```
Comment (kitsonk) [inline tools/cut_a_release.md:10]: This is not what @lucacasonato was saying.  Can we clarify this.

File: tools/cut_a_release.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Cutting a Deno release
+
+**During this process `main` branch should be frozen and no commits should land
+until the release is cut.**
```
Comment (kitsonk) [inline tools/cut_a_release.md:4]: Are there differences to account for versus creating a main release versus a branch patch release?

File: tools/cut_a_release.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+If this is a Deno patch release then the crates should have patch version bumped
+as well; analogously for a minor Deno release the crates should have minor
+version bumped.
+
+2. Make sure CI pipeline passes.
+
+3. Publish all bumped crates to `crates.io`
+
+This is done by running `cargo publish` in each crate, because of dependencies
```
Comment (kitsonk) [inline tools/cut_a_release.md:17]: A pre-req is that cargo is logged on with a user that has publish rights to all the crates.

File: tools/cut_a_release.md (hunk 1 above)
Comment (lucacasonato) [reply tools/cut_a_release.md]: Clarified it.

File: tools/cut_a_release.md (hunk 2 above)
Comment (bartlomieju) [reply tools/cut_a_release.md:4]: Added

File: tools/cut_a_release.md (hunk 3 above)
Comment (bartlomieju) [reply tools/cut_a_release.md:26]: Added a note

**Review by piscisaureus — APPROVED** [review]
LGTM, this is helpful.

**Pull request merged**